cloud/setup/kubernetes.tf

```diff
@@ -0,0 +1,119 @@
+resource "kubernetes_namespace" "kafka-producer-bench-ns" {
+  metadata {
+    name = "kafka-producer-benchmark"
+  }
+}
+
+resource "kubernetes_job" "producer-benchmark" {
+  depends_on = [confluent_kafka_topic.topics]
+  metadata {
+    name = "producer-benchmark"
+    namespace = kubernetes_namespace.kafka-producer-bench-ns.id
+  }
+  spec {
+    parallelism = var.producer_instances
+    template {
+      metadata {}
+      spec {
+        container {
+          name  = "producer-benchmark"
+          image = var.bench_producer_image
+          image_pull_policy = var.image_pull_policy
+          env {
+            name = "KAFKA_BOOTSTRAP_SERVERS"
+            value = confluent_kafka_cluster.perf-test-dedicated.bootstrap_endpoint
+            #value = "localhost:9092"
+          }
+          env {
+            name = "KAFKA_SECURITY_PROTOCOL"
+            value = "SASL_SSL"
+          }
+          env {
+            name = "KAFKA_SASL_MECHANISMS"
+            value = "PLAIN"
+          }
+
+          env {
+            name = "KAFKA_SASL_MECHANISM"
+            value = "PLAIN"
+          }
+          env { # GroupBy(key)
+            name = "AGG_PER_TOPIC_NB_MESSAGES"
+            value = "1000"
+          }
+          # Java specifics
+#          env {
+#            name  = "KAFKA_SASL_JAAS_CONFIG"
+#            value = "org.apache.kafka.common.security.plain.PlainLoginModule   required username='${confluent_api_key.perf-test-client-api-key.id}'   password='${confluent_api_key.perf-test-client-api-key.secret}';"
+#          }
```

Comment:
Should be uncommented

Reply:
I'd rather find a way to make this optional.
So that we can still target a locally running k8s Kafka instance.

Reply:
I have that in mind, let's keep that on the side for later.